games/devtest/mods/testnodes/param2.lua

```diff
@@ -253,7 +253,56 @@ core.register_node("testnodes:color4dir", {
 	groups = { dig_immediate = 3 },
 })
 
+minetest.register_node("testnodes:colorflowingliquid_source", {
+	description = S("Color Flowingliquid Source Test Node").."\n"..
+		S("param2 = color + 4 unused bits"),
+	paramtype2 = "color",
+	paramtype = "light",
+	liquidtype = "source",
+	liquid_alternative_source = "testnodes:colorflowingliquid_source",
+	liquid_alternative_flowing = "testnodes:colorflowingliquid_flowing",
+	liquid_renewable = true,
+	liquid_range = 8,
+	walkable = false,
+	buildable_to = true,
+	is_ground_content = false,
+	drawtype = "liquid",
+	palette = "testnodes_palette_flowingliquid.png",
```

Comment:
There is no such file, hence I can't easily test this properly. It seems like some changes are missing here?